.github/workflows/ci.yml

```diff
@@ -493,7 +494,7 @@ jobs:
       - go-apidiff
       - docker-checks
     steps:
-      - run: exit 0
+      - run: exit ${{ ((contains(needs.*.result, 'skipped') || contains(needs.*.result, 'failure')) && 1) || 0 }}
```

Comment:
> `( ... && 1) || 0 }`

💭 Funky syntax that reminds me of javascript.

❓ So, do I understand this correctly?

If no jobs skipped or failed:
- `((false || false) && 1) || 0`
- `(false && 1) || 0`
- `false || 0`
- `0`

If a job skipped:
- `((true || false) && 1) || 0`
- `(true && 1) || 0`
- `1 || 0`
- `1`

If a job failed:
- `((false || true) && 1) || 0`
- `(true && 1) || 0`
- `1 || 0`
- `1`

Reply:
Correct. I needed to experiment with the expression until I got it working. For example I first used ` ${{ !contains(needs.*.result, 'skipped') && !contains(needs.*.result, 'failure')) && 0 || 1 }`  but that didn't work because `0` is falsy